lib/src/app/idleTimer.js

```diff
@@ -8,29 +8,29 @@ import * as logger from 'loglevel';
 import castManager from './receiverManager';
 
 const timeout = {
-    PAUSED: 1000 * 60 * 5, // 5 minutes
-    IDLE: 1000 * 60 * 10 // 5 minutes
+  PAUSED: 1000 * 60 * 5, // 5 minutes
+  IDLE: 1000 * 60 * 10, // 5 minutes
```

Comment:
Dangling comma. There looks to be a new lint condition regarding this here:

https://github.com/ooyala/chromecast-sample-receiver/pull/61/files#diff-b0d0bf091f28af6d677bffef8898431bR4

Shouldn't it trigger on this?